libcontainer/cgroups/fscommon/utils.go

```diff
@@ -56,24 +56,24 @@ func ParseUint(s string, base, bitSize int) (uint64, error) {
 
 // ParseKeyValue parses a space-separated "name value" kind of cgroup
 // parameter and returns its key as a string, and its value as uint64
-// (ParseUint is used to convert the value). For example,
+// (using [ParseUint] to convert the value). For example,
 // "io_service_bytes 1234" will be returned as "io_service_bytes", 1234.
 func ParseKeyValue(t string) (string, uint64, error) {
-	parts := strings.SplitN(t, " ", 3)
-	if len(parts) != 2 {
+	key, val, ok := strings.Cut(t, " ")
+	if !ok {
```

Comment:
Not necessarily for this PR, but we could check for empty values here as well to return a nicer error than "strconv.ParseUint failed to parse ...";

```go
if !ok || key == "" || val == "" {
```